app/models/job.rb

```diff
@@ -36,6 +36,19 @@ class Job
   attr_accessor :script
   attr_accessor :state
 
+  # Handle the k and m suffix
+  attr_reader :min_nodes
```

Comment:
The `attr_accessor :min_nodes` line above this one should be removed.

Reply:
https://github.com/openflighthpc/flight-scheduler-controller/pull/2/commits/6ff71860dc4ceb1860e7156c3ab83b952b66ab5f